WordPress/Classes/Models/Plan.swift

```diff
@@ -118,7 +118,7 @@ struct PlanFeature {
     let slug: String
     let title: String
     let description: String
-    let iconName: String
+    let iconURL: NSURL?
```

Comment:
Not that I've seen, but `NSURL(string:)` is failable, so if a badly-formed URL was ever returned we'd need to be able to store it as nil?